docs/language/operators/from.md

```diff
@@ -54,20 +66,105 @@ from ... | switch color (
 ) | ...
 ```
 
-The output of a fork consists of multiple legs that must be merged.
-If the downstream operator expects a single input, then the output legs are
-merged with an automatically inserted [combine operator](combine.md).
+### Input Data
+
+Examples below below assume the existence of the Zed lake created and populated
+by the following commands:
+
+```mdtest-command
+export ZED_LAKE=example
+zed -q init
+zed -q create coinflips
+echo '{flip:1,result:"heads"} {flip:2,result:"tails"}' | zed -q -use coinflips load -
+zed -q -use coinflips branch trial 
+echo '{flip:3,result:"heads"}' | zed -q -use coinflips@trial load -
+zed -q create numbers
+echo '{number:1,word:"one"} {number:2,word:"two"} {number:3,word:"three"}' | zed -q -use numbers load -
+zed ls | awk '{ print $1 }' | sort
```

Comment:
I discovered late in the game that this `zed ls` output was not deterministic so sometimes the test would fail when the pool names came back in reverse order, hence the `sort`. Knowing this is metadata queried from the pool and how pool queries work, I wasn't altogether surprised by this. However, since this is user-facing data and it's not necessary for users to know how it's generated, it made me think that it might be appropriate for us to sort these automatically so it comes back deterministic, at least when requested via `zed`. Probably not a big deal regardless, but if anyone sees this comment & agrees I'll happily open up an issue so we can get to it later.